### PR TITLE
chore(docs): migrate vuepress to vitepress

### PR DESCRIPTION
**Pull request by Sepush**
Signed-off-by: Sepush <sepush@outlook.com>

### CODE REVIEW

**Comment by imkiva** [comment]
Can we still have the banner image filling all the width like https://www.aya-prover.org/ ?

**Comment by Sepush** [comment]
> Can we still have the banner image filling all the width like https://www.aya-prover.org/ ?

I should do something to achive this

**Review by imkiva — COMMENTED** [review]
vitepress seems to not respect accent color defined in `.vitepress/styles/palette.styl` 🤔

**Comment by imkiva** [comment]
> > Can we still have the banner image filling all the width like https://www.aya-prover.org/ ?
> 
> I should do something to achive this also for the font and latex

Oh! Thank you!!

File: src/.vitepress/config.ts [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+      crossorigin: "anonymous",
+    }],
+    ['link', {
+      rel: 'stylesheet',
+      href: "https://fonts.googleapis.com/css2?family=Merriweather:ital,wght@0,300;0,700;0,900;1,300;1,700&display=swap",
+    }],
+    ['link', {
+      rel: 'icon',
+      href: `/static/img/logo.svg`,
+    }],
```
Comment (imkiva) [inline src/.vitepress/config.ts:20]: Also `/logo.svg` ?

Comment (Sepush) [reply src/.vitepress/config.ts:20]: yes I miss this one fix it later

**Comment by Sepush** [comment]
I design a new home page and new theme

**Comment by Sepush** [comment]
image
ready

**Comment by ice1000** [comment]
> I design a new home page and new theme 
> image
> image
> 

Can we instead make the background image to work? :sob:

**Comment by Sepush** [comment]
> > I design a new home page and new theme
> > image
> > image
> 
> Can we instead make the background image to work? 😭

I you prefer the origin background I will try to make it work

**Pull request merged**